tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -258,6 +258,41 @@ describe('Service Form Modal', function () {
       cy.visitUrl({url: '/services/overview/%2F/create'});
     });
 
+    it('should fill the entire viewport', function () {
+      var isModalFullScreen = true;
+
+      cy.window()
+        .then(function ($window) {
+          cy.get('.modal-full-screen')
+            .should(function ($element) {
+              if ($element[0].clientHeight !== $window.innerHeight || $element[0].clientWidth !== $window.innerWidth) {
+                isModalFullScreen = false;
+              }
+              expect(isModalFullScreen).to.be.equal(true);
+            });
+        });
+    });
+
+    it('should be horizontally and vertically centered in the modal container', function () {
+      cy.get('.modal-body-wrapper')
+        .should(function ($modalWrapper) {
+          const modalWrapperRect = $modalWrapper[0].getBoundingClientRect();
+          const modalWrapperTop = Math.floor(modalWrapperRect.top + (modalWrapperRect.height / 2));
+          const modalWrapperLeft = Math.floor(modalWrapperRect.left + (modalWrapperRect.width / 2));
+
+          const $modalContent = $modalWrapper.find('.row.panel-grid');
+          const modalContentRect = $modalContent[0].getBoundingClientRect();
+          const modalContentTop = Math.floor(modalContentRect.top + (modalContentRect.height / 2));
+          const modalContentLeft = Math.floor(modalContentRect.left + (modalContentRect.left / 2));
```

Comment:
Same here, let's rename these variables to reflect the fact that we're finding the mid-point of these elements.